src/components/VSelectize.vue

```diff
@@ -264,7 +269,7 @@ export default {
      * Action performed when component is focused
      */
     onFocus () {
-      this.hasFocus = true
+      this.hasFocus = !this.disabled
       this.$refs.input.focus()
```

Comment:
Since the input (selectize-componet) is disable it should not allow focus at all, please put the  on focus code inside a condition that if `this.disabled` nothing should happen and else should proceed the normal opeation